src/form/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import cache from '@dojo/framework/core/middleware/cache';
+import theme from '@dojo/framework/core/middleware/theme';
+import { RenderResult } from '@dojo/framework/core/interfaces';
+
+import createFormMiddleware, { FormMiddleware } from './middleware';
+import * as css from '../theme/form.m.css';
+
+const form = createFormMiddleware();
+
+interface FormProperties {
+	/** The initial form value */
+	initialValue?: Record<string, any>;
```

Comment:
I wonder if this should be `type FormValue = Record<string, any>` so it's repeated in here a bunch of times. Not sure if we'll ever need to change the type, but if so it might make lives easier!